reference/api/multi_search.mdx

```diff
@@ -2,7 +2,9 @@
 
 The `/multi-search` route allows you to perform multiple search queries on one or more indexes by bundling them into a single HTTP request. Multi-search is also known as federated search.
 
-## POST route
+## Perform a multi-search
+
+Bundle multiple search queries in a single API request.
```

Comment:
```suggestion
Bundle multiple search queries in a single API request. Multi-search queries do not need to belong to the same index. You can search through multiple indexes at once using this endpoint.
```

WDYT @guimachiavelli ?

Reply:
I find it a bit weird to speak of queries _belonging_ to an index, to be honest. Perhaps we can just skip that sentence altogether?

```suggestion
Bundle multiple search queries in a single API request. Using this endpoint to search through multiple indexes at once.
```